web/virtual_lab/static/virtual_lab/js/physics_projectile.js

```
@@ -0,0 +1,412 @@
+    v0 = Math.hypot(dx_m, dy_m);
+    thetaRad = Math.atan2(dy_m, dx_m);
+    vx = v0 * Math.cos(thetaRad);
+    vy0 = v0 * Math.sin(thetaRad);
+
+    // Compute theoretical max range & height
+    maxRange = (v0 * v0 * Math.sin(2 * thetaRad)) / g;
+    maxHeight = (v0 * v0 * Math.sin(thetaRad) * Math.sin(thetaRad)) / (2 * g);
+
+    // Recalculate pixelsPerMeter so full path fits
+    const marginX = 60, marginY = 60;
+    const availableWidth = canvas.width - marginX;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)